tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -126,9 +126,20 @@ kubetest2 kops "${KUBETEST2_ARGS[@]}" \
   --create-args="${create_args[*]}" \
   --control-plane-size="${KOPS_CONTROL_PLANE_SIZE:-1}"
 
+KUBECONFIG=$(mktemp -t kubeconfig)
```

Comment:
could you add the `.XXXXX` suffix like the other usages?
https://cs.k8s.io/?q=mktemp%20-t&i=nope&files=&excludeFiles=&repos=